static/js/course_list.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+/* global SETTINGS:false */
```

Comment:
can you add `__webpack_public_path__` at the top of this file? you can take a look at a bunch of the other entry points (e.g. `static/js/dashboard.js`) for an example.